animalgame.js

```diff
@@ -13,13 +13,18 @@ function Node(data, y, n) {
 
 var readlineSync = require('readline-sync');
 var fs = require("fs");
+//add some encouragement//
+var words = ["That's great!", "You know your animals!", "Let's play again!", "One more animal!"];
+var word = words[Math.floor(Math.random() * words.length)];
```

Comment:
You might consider picking the random "phrase" below on line 51 so it's different each game in the same session!